sources/packages/web/src/components/aest/InstitutionUserSummary.vue

```diff
@@ -73,11 +72,25 @@
         sortable="true"
       ></Column>
       <Column :field="UserFields.Email" header="Email" sortable="true"></Column>
-      <Column :field="UserFields.UserType" header="User Type"></Column>
+      <Column :field="UserFields.UserType" header="User Type">
+        <template #body="slotProps">
+          <ul
+            class="no-bullets"
```

Comment:
mm, we do have bootstrap class `list-unstyled`, to achieve this 
![image](https://user-images.githubusercontent.com/77353155/149831641-8ce8668b-297a-4c2d-866a-fa3be25ca8e6.png)

Reply:
Retaining the custom css class for the sake of alignment.